src/boot/lexer.mll

```diff
@@ -30,7 +30,6 @@ let reserved_strings = [
   ("let",           fun(i) -> Parser.LET{i=i;v=()});
   ("recursive",     fun(i) -> Parser.REC{i=i;v=()});
   ("lam",           fun(i) -> Parser.LAM{i=i;v=()});
-  ("fix",           fun(i) -> Parser.FIX{i=i;v=()});
```

Comment:
We should also remember to remove `fix` from the MCore parser.

Reply:
I quite forgot to look at that. Is it just changes in `meta.mc`? What's praxis for testing it?

Reply:
Until we've sorted out having multiple `use`s of the same language, you should change it in `meta.mc` and `mcore_parser.mc`. The latter you can test by feeding it the test files in `test/mexpr` (this is currently not done automatically).

Reply:
I've now pushed updates that make `meta.mc` and `mcore_parser.mc` both use recursive lets and parse programs that contain them.